File: tests/swift/input.swift
class Solution {
    func containsDuplicate(_ nums: [Int]) -> Bool {
        var sortedNums = nums.sorted()
        for i in 1..<sortedNums.count {
            if sortedNums[i] == sortedNums[i - 1] {
                return true
            }
        }
        return false
    }
}

File: tests/php/input.php
class Solution {
    function containsDuplicate($nums) {
        sort($nums);
        $n = count($nums);
        for ($i = 1; $i < $n; $i++) {
            if ($nums[$i] == $nums[$i - 1]) {
                return true;
            }
        }
        return false;
    }
}